scripts/validate-models-s3.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+strategies=${STRATEGIES:=fast,hi_res,ocr_only}
```

Comment:
How about passing these as arguments so we can reuse this script for other strategies? If this sounds good, documenting how to use the script would help too.

Reply:
In essence this solves the same thing by allowing you to set the env var to the values you want. This just sets the default in case one is not set.

Reply:
That makes sense. Commenting or documenting in the script that the user can specify strategies by setting the STRATEGIES environment variable makes it easier to use for someone who isn't familiar with shell scripting.